js/classes/entities/midground/enemies/yeti.js

```diff
@@ -87,20 +90,32 @@ class Yeti {
         this.removeFromWorld = true;
 
         // add a piece of bacon in the Yeti's place at bottom-center of yeti
-        const baconPos = new Vector(this.pos.x, this.pos.y + Yeti.SCALED_SIZE.y / 2);
-        GAME.addEntity(new FoodDrop(FoodDrop.BACON, baconPos));
+        const pos = this.base.getCenter();
+        GAME.addEntity(new FoodDrop(pos, FoodDrop.GIANT_MUSHROOM));
```

Comment:
XD this is its drop? One thing to keep in mind. While I love this we need to make sure it can get used. So either we should put this in inventory for use later and give it a description saying it is rare and to spare it or put enemies near by that are hugely difficult because of quantity so this can be used immediately. Just some thoughts. Curious if you have thoughts on this @NathanHinthorne.